exercises/node-javascript/add-esdk-complete/retrieve.js

```diff
@@ -1,25 +1,21 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-/* ADD-ESDK-START:
- * Add the @aws-crypto/client-node dependency.
- * You will need the decrypt stream as well as the KMS keyring.
- * Look at the config file to see how to pull the Faythe CMK.
- * Then pipe the encrypted stream to the decrypt stream: Profit.
- */
-
+// ADD-ESDK-START: Add the dependency
 const { decryptStream, KmsKeyringNode } = require("@aws-crypto/client-node");
 const { S3 } = require("aws-sdk");
 const s3 = new S3();
 const config = require("./config");
 const Bucket = config.state.bucketName();
 
+// ADD-ESDK-START: Add the dependency
```

Comment:
// ADD-ESDK-START: Set up a keyring to use Faythe's CMK for decrypting.

Reply:
// ADD-ESDK-START: Plumb In Your Config

That matches the instructions.  Pushed.